crates/api_common/src/community.rs

```diff
@@ -37,12 +37,14 @@ pub struct CreateCommunity {
   pub banner: Option<String>,
   pub nsfw: Option<bool>,
   pub posting_restricted_to_mods: Option<bool>,
+  pub discussion_languages: Option<Vec<LanguageId>>,
```

Comment:
I dont think its necessary to copy all fields from EditCommunity here if no api client actually uses it. Also it looks like this is currently not used at all.

Reply:
Its better for consistency; either we duplicate every field ( which is fine because many are optional anyway ), or we come up with a system of rules for which fields are only on one, and why.

Reply:
Okay, but then you also need to pass this field to the db in the api handler.

Reply:
k done